LeaderboardBackend/Controllers/AccountController.cs

```diff
@@ -114,4 +116,51 @@ public async Task<ActionResult<LoginResponse>> Login([FromBody] LoginRequest req
             badCredentials => Unauthorized()
         );
     }
+#pragma warning restore CS1573
+
+    /// <summary>
+    ///     Resends the account confirmation link.
+    /// </summary>
+    /// <param name="authService">IAuthService dependency.</param>
+    /// <param name="confirmationService">IAccountConfirmationService dependency.</param>
+    /// <param name="emailSender">EmailSender dependency.</param>
+    /// <response code="200">The request was sent successfully.</response>
```

Comment:
Should this be more descriptive? You could write something like "A new confirmation link was generated."